templates/layout.html

```diff
@@ -5,20 +5,20 @@
       <title>Template</title>
    </head>
 <html>
-	<body>
-		<header>
-			<div class="container">
-			<strong><nav>
-				<ul class="menu">
-					<li><a href="{{ url_for('Home') }}"> Home </a></li>
-					<li><a href="{{url_for('contact_us') }}">contact_us</a></li>
-				</ul>
-			</nav></strong>
-			</div>
-		</header>
-		<div class="container">
-			{% block content %}
-			{% endblock %}
-		</div>
-	</body>
-</html>
\ No newline at end of file
+    <body>
+        <header>
+            <div class="container">
+            <strong><nav>
+                <ul class="menu">
+                    <li><a href="{{ url_for('home') }}"> Home </a></li>
+                    <li><a href="{{url_for('contact_us') }}">contact_us</a></li>
+                </ul>
+            </nav></strong>
+            </div>
+        </header>
+        <div class="container">
+            {% block content %}
+            {% endblock %}
+        </div>
+    </body>
+</html>
```

Comment:
@SaarCohen93 Please try to stick to one indentation style: either tabs or spaces. I suggest to use space instead of tabs as majority of the files you committed are indented with spaces. Also python style guide [PEP8](https://www.python.org/dev/peps/pep-0008/#indentation) suggests to use 4 spaces.

Reply:
This is an old file, I deleted it.

Reply:
Not resolved yet.